english/Pokemon_Go/Pokemon-Go.md

```diff
@@ -0,0 +1,6 @@
+Pokemon Go has seemingly taken over the world. You want to be the very best, so
+you quit your job to become a full time Pokemon Go player. You strive to travel
+the world in search of Pokemon!
+
+One night, after quitting your job, you are dreaming of
+[Marshmallows](../pikachu/Pokeball.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.